src/linear_algebra/determinant.lean

```diff
@@ -368,6 +368,27 @@ end
 lemma basis.is_unit_det (e' : basis ι R M) : is_unit (e.det e') :=
 (is_basis_iff_det e).mp ⟨e'.linear_independent, e'.span_eq⟩
 
+/-- Any alternating map to `R` where `ι` has the cardinality of a basis equals the determinant
+map with respect to that basis, multiplied by the value of that alternating map on that basis. -/
+lemma alternating_map.eq_smul_det (f : alternating_map R M R ι) : f = f e • e.det :=
+begin
+  rw ←alternating_map.coe_multilinear_map_injective.eq_iff,
+  refine basis.ext_multilinear e _,
```

Comment:
Can you extract these two lines to `basis.ext_alternating`?

Reply:
I think it's useful to have simply because it consumes two or three lines of this proof. We can always rename it when we have the stronger version available.

Reply:
I've added that lemma.